v2.0/insert.md

```diff
@@ -11,7 +11,8 @@ The `INSERT` [statement](sql-statements.html) inserts one or more rows into a ta
 ## Performance Best Practices
 
 - A single [multi-row `INSERT`](#insert-multiple-rows-into-an-existing-table) statement is faster than multiple single-row `INSERT` statements. To bulk-insert data into an existing table, use a multi-row `INSERT` instead of multiple single-row `INSERT` statements.
-- The experimental <a href=import.html><code>IMPORT</code></a> statement performs better than <code>INSERT</code> when inserting rows into a new table. 
+- The experimental <a href=import.html><code>IMPORT</code></a> statement perfo
```

Comment:
Not sure why the line is wrapping mid-word. While you're editing this page, can you fix that. Also, it'd be great if you could replace the html for Markdown:

```
The experimental [`IMPORT`](import.html) statement performs better than `INSERT` when inserting into a new table.
```